app/controllers/bookings_controller.rb

```diff
@@ -2,9 +2,20 @@
 class BookingsController < ApplicationController
   before_action :logged_in_user, :load_bookings, only: %i(index destroy)
   before_action :check_booking_owner, :is_allow_cancel_booking, only: :destroy
+  before_action :logged_in_user, :load_field_type, :has_overlap, only: :create
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i(index destroy create)
  before_action :load_bookings, only: %i(index destroy)
  before_action :check_booking_owner, :is_allow_cancel_booking, only: :destroy
  before_action :load_field_type, :has_overlap, only: :create
```
Thà để method ra 1 line riêng, chứ không được lặp lại method 2 lần trên before_action